draft/7/4d-projection-and-basic-hypercube-twists/index.md

```diff
@@ -0,0 +1,133 @@
+| `B`          | Back       | negative Z | Back           |
+| `O`          | Out        | positive W | Pink           |
+| `I`          | In         | negative W | Purple         |
+
+Note: historically, `K` ("kata") has been used instead of `I`, and `A` ("ana") or `T` ("top") have been used instead of `I`. It is recommended to avoid these because it is confusing which one corresponds to which cell. Furthermore, `A` conflicts with 5D notation (`A` for "anterior") and `T` has been reappropriated for general F2L notation (`T` for general "top").
+
+These also define the standard projection:
+
+- Positive X pointing to the right
+- Positive Y pointing up
+- Positive Z pointing forward toward the 3D camera
+- Positive W pointing forward toward the 4D camera
+
+The `O` facet is typically hidden, but the other seven facets are typically visible and spaced apart. The default 3D camera may have a rotational offset applied to give a better view of the 7 facets as follows:
+
+- `R` may point more forward and down
+- `F` may point more left and down
+- `U` may point more forward
+
+## 7.2 Clockwise Twists
+
+Twists are named using 2, 3, or 4 letters, each corresponding to a facet. All letters are uppercase, with no space or punctuation between them. The first letter indicates which facet is rotated; the remaining letters describe the polytope element that is fixed by the twist. Letters other than the first one may be specified in any order. The twist occurs in a 2D plane, over the minimum nonzero angle necessary to restore the puzzle to its original shape.
+
+| Twist type   | Number of letters | Angle | Example |
+| ------------ | ----------------- | ----- | ------- |
+| Ridge twist  | 2                 | 90°   | `IR`    |
+| Edge twist   | 3                 | 180°  | `OUR`   |
+| Corner twist | 4                 | 120°  | `DUFR`  |
+
+Rotations appear clockwise when the 4D camera is facing along the vector of the first facet and the 3D camera is facing along the _opposite_ vector from the polytope element described by the second facet. For example, the twist `IR` twists a layer of the `I` facet from positive Z to positive Y, which keeps the vector along the X axis fixed.
+
+Examples:
+
+| Twist name | Description                                                              |
+| ---------- | ------------------------------------------------------------------------ |
+| `IR`       | `R` twists 90° from `F` to `U`                                           |
+| `OUR`      | `O` twists 180°, such that `U` and `R` swap and `F` and `B` swap         |
+| `DUFR`     | `D` twists 120° clockwise, such that `F` goes to `I` and `I` goes to `R` |
+|            |                                                                          |
+
+In software with mouse controls, these twists are typically executed by right-clicking on the described sticker. For example, `DUFR` is executed by right-clicking on the `D` sticker of the `DUFR` corner piece.
```

Comment:
Hmm, we also use that convention but that sounds a bit out of scope for this spec?

Reply:
It's technically possible to describe 4D cube moves unambiguously without also introducing piece names and mouse controls, but realistically no one will understand this unless we introduce them all at once. Each move is basically named for the sticker you click to execute it, and that sticker stays fixed during the rotation.